custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,146 @@
+                        # 1 Byte = 8 bits
+                        # 1 byte is equal to 0.001 kilobytes
+                        KBs = rate / 1000
+                        # Kbs = KBs * 8
+                        value = KBs
+
+                    new_property = f"{prop_name}_{label}"
+                    interface[new_property] = int(round(value, 0))
+
+            for server_name in dict_get(
+                self._state, "telemetry.openvpn.servers", {}
+            ).keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)